packages/trader/src/Modules/Trading/Components/Form/ContractType/ContractTypeMenu/contract-type-menu.tsx

```diff
@@ -113,14 +121,21 @@ const Dialog = ({
                                         <div className='dc-vertical-tab__content-container'>
                                             {selected_category_contract && <NoResultsMessage text={input_value} />}
                                             {info_banner}
+                                            {learn_more_banner}
                                             {renderChildren()}
                                         </div>
                                     </div>
                                 </VerticalTab.Layout>
                             ) : (
                                 <React.Fragment>
                                     <div className='dc-vertical-tab__action-bar dc-vertical-tab__action-bar--contract-type-info-header'>
-                                        <Header title={item.text || ''} onClickGoBack={onBackButtonClick} />
+                                        <Header
+                                            title={localize(title || item.text || '')}
+                                            onClickGoBack={onBackButtonClick}
+                                            onClickCross={onClose}
+                                            should_render_arrow={!hide_back_button}
+                                            should_render_cross={hide_back_button}
```

Comment:
```suggestion
                                            has_arrow={!hide_back_button}
```
let's unite `should_render_arrow` and `should_render_cross` props into one since they are the opposites of each other. this way, if has_arrow is true we will show arrow, otherwise - we will show the Close icon (better to call it a Close icon, not cross). :)

Reply:
No, I think it's better not to do so, because <Header /> component is used not only in this file. In some cases, there should be only arrow, in others - only close (cross), the rest - both.
But will rename